pkgs/top-level/python-packages.nix

```diff
@@ -7405,8 +7403,6 @@ in {
 
   sybil = callPackage ../development/python-modules/sybil { };
 
-  symbiflow-fasm = callPackage ../development/python-modules/symbiflow-fasm { };
```

Comment:
Can we leave those packages in?

Reply:
I think only textx is usefull without Symbiflow and project XRay

Reply:
No. Please read my actual motivation beforehand, since you seemed to have missed it (I linked it in the PR description). Individual packages like `symbiflow-fasm` can go elsewhere (e.g. in my proposed separate repository, or NUR), and if  CajuM/Mihai wants to re-add some of the low-maintenance packages to upstream nixpkgs, I *might not be opposed to that, on a case by case basis.*

In general, I'm not convinced of the merit of these packages, all other technical considerations aside. So I'm not inclined to leave them.

Reply:
And yes, of all the packages, only `textx` seemed more generally useful at a glance. I could be convinced of others though, but that one I think could be added (that said I'm not sure how popular it actually *is*...)